tutorials/hijack-escrow/hack-contract.md

```diff
@@ -56,226 +46,167 @@ From `msg.rs`:
+    /// Once an escrow is expired, it can be returned to the original funder (via "refund").
+    pub expiration: Option<Expiration>,
 }
 ```
 
-Note that we use `Addr`, which is a validated address wrapper with some helper functions for storage inside `State
-`, while we use invalidated  `String` address which should be validated by developer, for messages and anything that
-interacts with the user. There is [more info on addresses here](https://docs.cosmwasm.com/0.14/architecture/addresses).
-
-`Option<u64>` is a way of telling rust this field may be missing. It may either have a value, like `Some(123456)` or
-be `None`. This means the init message may omit those fields (or pass them as `null`) and we don't need to use some
-special value like `0` to signify disabled.
+Note that we use the type `Addr`, which is a validated address wrapper with some helper functions, for addresses stored inside the `Config`, while we use invalidated `String` addresses which should be validated by the contract, for messages sent and received through user interaction. There is [more info on addresses here](https://docs.cosmwasm.com/1.0/architecture/addresses).
 
-Moving to the `ExecuteMsg` type, which defines the different contract methods, we make use of a slightly more complex
-rust construction, the [`enum`](https://doc.rust-lang.org/stable/rust-by-example/custom_types/enum.html). This is also
-known as [a tagged union or sum type](https://en.wikipedia.org/wiki/Tagged_union), and contains a fixed set of defined
-possible data types, or `variants`, *exactly one of which must be set*. We use each `variant` to encode a different
-method. For example `Execute::Refund{}` is a serializable request to refund the escrow, which is only valid after a
-timeout.
+`Option<Expiration>` is a way of telling rust this field may be missing. It may either have a value, like `Some(at_height: 1230405)` or be `None`. This means the init message may omit those fields (or pass them as `null`) and we don't need to use some special value like `0` to signify they are disabled.
 
+Moving to the `ExecuteMsg` and `QueryMsg` types, which define the different contract methods, we make use of a slightly more complex rust construction, the [`enum`](https://doc.rust-lang.org/stable/rust-by-example/custom_types/enum.html). This is also known as [a tagged union or sum type](https://en.wikipedia.org/wiki/Tagged_union), and contains a fixed set of defined possible data types, or `variants`, *exactly one of which must be set*. We use each `variant` to encode a different method. For example `Execute::Refund{}` is a serializable request to refund the escrow, which can only be valid after a timeout.
 ```rust
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum ExecuteMsg {
-  Approve {
-    // release some coins - if quantity is None, release all coins in balance
-    quantity: Option<Vec<Coin>>,
-  },
-  Refund {},
+    Approve {
```

Comment:
Your code editor indentation auto applies these spaces. Best to follow the documentation practices.